google/cloud/spanner_dbapi/cursor.py

```diff
@@ -232,13 +243,18 @@ def description(self):
 
     @property
     def rowcount(self):
+        """Count number of rows in the table after last execution.
```

Comment:
Wrong! See the `:returns:` as well as [PEP-249](https://www.python.org/dev/peps/pep-0249/#rowcount)